vita3k/modules/SceIme/SceIme.cpp

```diff
@@ -17,12 +17,32 @@
 
 #include "SceIme.h"
 
+#include <ime/types.h>
+
+#include <kernel/thread/thread_functions.h>
+#include <util/lock_and_find.h>
+
 EXPORT(int, sceImeClose) {
-    return UNIMPLEMENTED();
+    host.ime.ime_state = false,
+    memset(&host.ime.ime_param, 0, sizeof(SceImeParam));
+    LOG_DEBUG("sceImeClose");
```

Comment:
Delete

Reply:
Off course not worry is only for see when is called is only for debug ^^